.env.example

```
@@ -1,3 +1,6 @@
 EXPO_PUBLIC_API_URL="http://localhost:8081"
 EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY="Your Expo Public Clerk Publishable Key Here"
+# keep Clerk secret key secure: do not expose it to client-side code
+CLERK_SECRET_KEY="<YOUR_CLERK_SECRET_KEY>"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)